op-e2e/actions/upgrades/utils/assertions.go

```diff
@@ -0,0 +1,31 @@
+package utils
+
+import (
+	"math/big"
+
+	actionhelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/interop/dsl"
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/lmittmann/w3"
+	"github.com/stretchr/testify/require"
+)
+
+var ProxyImplGetterFunc = w3.MustNewFunc(`implementation()`, `address`)
+
+func RequireUpgradeSuccessful(t actionhelpers.Testing, actors *dsl.InteropActors, implAddr common.Address, proxyAddress common.Address, activationBlockNumber *big.Int) {
+	code, err := actors.ChainA.SequencerEngine.EthClient().CodeAt(t.Ctx(), implAddr, activationBlockNumber)
+	require.NoError(t, err)
+	require.NotEmpty(t, code, "CrossL2Inbox contract should be deployed")
```

Comment:
```suggestion
	require.NotEmpty(t, code, "Contract should be deployed")
```
Or you could accept the contract name as a function argument and shove it in here.

Reply:
Good catch, thanks

Reply:
Fixed in a58d5663930df0f6ba41b5d8d8c9b245da80c0bd.